### PR TITLE
Added overview from back end repo to keep it consistent

### PR DESCRIPTION
**Pull request by Donovon**
Keeping docs consistent with the back end repo!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,56 +48,4 @@ You can use a tool like [nvm](https://github.com/nvm-sh/nvm) to get the version
...
-
-### `npm lint:css`
-
-Run [stylelint](https://stylelint.io/) to check for any problems in your CSS code.
-
-## Learn More
-
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
-
-To learn React, check out the [React documentation](https://reactjs.org/).
```
Comment (BrandonBolton) [inline README.md:70]: I believe we should leave this in. It a nice bit of information for a quick start on contributing.

**Review by BrandonBolton — CHANGES_REQUESTED** [review]

**Review by BrandonBolton — APPROVED** [review]
Just saw the message about removing the npm scripts. Everything is good to go I think!

File: README.md (hunk 1 above)
Comment (jrapala) [reply README.md:70]: All we need at the moment is 'npm start' which is mentioned earlier. 

We can adjust the docs as we add more scripts.

Comment (Donovon) [reply README.md:70]: I could go either way - Juliette suggested removing it in Slack.

Might be a bit much for the README - but we could have a second file for setup instructions?  I know that's on our list of things to do in the short term.  Just an idea.

**Review by jrapala — APPROVED** [review]

**Pull request merged**